ui/desktop/src/components/alerts/AlertBox.tsx

```diff
@@ -49,7 +49,8 @@ export const AlertBox = ({ alert, className }: AlertBoxProps) => {
     loadThreshold();
   }, [read]);
 
-  const currentThreshold = loadedThreshold !== null ? loadedThreshold : alert.autoCompactThreshold;
+  const currentThreshold =
+    loadedThreshold !== null ? loadedThreshold : (alert.autoCompactThreshold ?? 0.8);
```

Comment:
can we just make loadedThreshold not optional and initialize to 0.8? also delete autoCompactThreshold, it doesn't seem like we're using it (maybe we used to and that's how this broke?)